index.html

```diff
@@ -315,5 +309,13 @@ <h1>Make a Pledge:</h1>
     document.location.href="https://twitter.com/intent/tweet?text=" +encodeURI(text)+ "&url=" + encodeURI(url);
   }
   
+
+  document.getElementById('hardcover').onclick = function(e){
+    document.location.href="https://www.amazon.com.br/Infinite-Machine-Crypto-hackers-Building-Internet/dp/0062886142/ref=tmm_hrd_swatch_0?_encoding=UTF8&qid=1644880269&sr=8-7"
+  }
+  
+  document.getElementById('epub').onclick = function(e){
+    document.location.href="https://www.amazon.com.br/Infinite-Machine-Crypto-hackers-Building-Internet-ebook/dp/B07X8HS2WC/ref=tmm_kin_swatch_0?_encoding=UTF8&qid=1644880269&sr=8-7"
```

Comment:
these are the wrong lniks.  pls defer to whatever is in the a href

Reply:
fixed to  document.location.href="#"